pypesto/ensemble/utils.py

```diff
@@ -230,3 +237,57 @@ def get_prediction_dataset(ens: Union[Ensemble, EnsemblePrediction],
                         'an EnsemblePrediction object as input. Stopping.')
 
     return dataset
+
+
+def read_ensemble_prediction_from_h5(
+        predictor: Union[Callable[[Sequence], PredictionResult], None],
+        input_file: str):
+
+    # open file
+    with h5py.File(input_file, 'r') as f:
+        pred_res_list = []
+        bounds = {}
+        for key in f.keys():
```

Comment:
Not sure, can the following be used, to simplify the code block a bit?
```python
for key, value in f.items():
   ...
```

Reply:
sadly no, as the values then are `<HDF5 dataset "lower_bound": shape (2,), type "<f8">`.